### ISSUE TITLE
support redrock

### ISSUE THREAD

**Issue by Kumassy** [Kumassy/ownserver-client-gui#21]

**Comment by rouyamawolf** [comment]
マイクラ統合版でのポート開放はこのソフトでできるのでしょうか？
できるのであれば教えてくださると幸いです。

**Comment by Kumassy** [comment]
コメントありがとうございます。

「カスタム設定」から Bedrock 版のサーバーを立ち上げることができるはずですが、 Windows 環境ではバグが見つかったので現時点では利用できません...。

この問題は認識していて、 https://github.com/Kumassy/ownserver-client-gui/issues/25 で作業します。

本件になにかアップデートがあればこちらでも共有します

**Comment by rouyamawolf** [comment]
ありがとうございます！！！！
すいません絶対パスって何なんでしょうか...?

**Comment by rouyamawolf** [comment]
![image](https://github.com/Kumassy/ownserver-client-gui/assets/132792038/9167aa41-b982-42e6-a798-02e24b9b0702)
一応できたのですがエラーが起きて使えないです・・・どうすればよいのでしょうか

**Comment by Kumassy** [comment]
port [19132] may be in use by another process となっているので、裏で bedrock_server.exe が起動していてサーバーが二重で動いていると思います。

一度Windowsごと再起動してみてください

**Comment by rouyamawolf** [comment]
![image](https://github.com/Kumassy/ownserver-client-gui/assets/132792038/cd9ed0c9-ab82-4681-a81c-d2c466f8afc4)
再起動して初回起動なんですがこうなってしまいました・・・どうすればいいですかね・・・

**Comment by Kumassy** [comment]
同じエラーが出ていますね。

OwnServer を使わずに、エクスプローラから bedrock_server.exe をダブルクリックしてみてください。
コマンドプロンプトが立ち上がって Minecraft のサーバーが起動するはずです。

もし `port [19132] may be in use by another process` というエラーが出るようならば OwnServer の問題ではなく、お使いの環境に問題があります。

**Comment by Kumassy** [comment]
> あ、もしかしてbedrock_server.exe起動せずにownserver使う感じですか？

そうですね、 「ゲームサーバーの起動」画面で「起動」をクリックすると、 bedrock_server.exe を自動的に起動してくれます。
別途 bedrock_server.exe を起動すると二重起動になってしまいます


> IPはどこ見ればいいのでしょうか！

こちらのページを参照いただけますか？
java 版ですが、後半の 「 OwnServer の設定」からは共通です
https://ownserver.kumassy.com/games/minecraft

**Comment by rouyamawolf** [comment]
ありがとうございます！！！

2023年5月23日(火) 午後1:00 Kumassy ***@***.***>:

**Comment by rouyamawolf** [comment]
度々すいません！
一応ポート開放までできてIPコピーして入ろうとしたら無効なIPですっていわれました
どうすればいいですか！

**Comment by Kumassy** [comment]
切り分けとして、 OwnServer を使わずに bedrock server に直接接続できるかを調べたいです。

OwnServer を使わず、 bedrock_server.exe をダブルクリックして bedrock server を起動した状態で
Minecraft で入力するアドレスを

```
IP: localhost
Port: 19132
```

としてください。これで接続できないようであれば Minecraft 側に問題があることになります。

**Comment by rouyamawolf** [comment]
接続できないんでMinecraft側に問題ありそうです・・・どうすればいいですかね・・・

2023年5月23日(火) 22:45 Kumassy ***@***.***>:

**Comment by Kumassy** [comment]
Minecraft 側の設定なのか、もしかすると Windows 自体の問題かもしれません。
いずれにしてもこちらでは対応は難しいので、もし解決策が見つかれば共有いただけるとたすかります...。

**Comment by rouyamawolf** [comment]
もしかしたら二重ルーターとかになってるかもしれないのでそこら辺の設定とかいじってみます...！

2023年5月26日(金) 午後8:41 Kumassy ***@***.***>:
